bin/deepstate/main_evm.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# Copyright (c) 2017 Trail of Bits, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+logging.basicConfig()
+
+import collections
+import sys
+import os
+import md5
+
+try:
+  import manticore.ethereum
+except Exception as e:
+  if "Z3NotFoundError" in repr(type(e)):  
+    print "Manticore requires Z3 to be installed."
+    sys.exit(255)
+  else:
+    raise
+import traceback
+from .common import DeepState
+
+L = logging.getLogger("deepstate.evm")
+L.setLevel(logging.INFO)
+
+def move_tests(workspace, outdir):
+  """Parse and sort testcases"""  
+  tx_files = []
+  for prefix,d,files in list(os.walk(workspace)):
+    for f in files: 
+      if "test" in f and ".tx" in f:
+        tx_files.append(prefix+"/"+ "/".join(d) + f)
```

Comment:
Use `os.path.join` and the `.format` method for strings in place of string concatenation.